src/components/PlayDetailsTab.module.scss

```diff
@@ -34,9 +34,9 @@
 }
 
 .content, .sidebar {
-  height: calc(100vh - (var(--x) * 6.5 + 1em));
+  height: calc(100vh - (8em - 1px) - (var(--bootstrap-padding) * 3));
```

Comment:
Can this calculation be explained a bit?

Reply:
Sure! 

Since sigma canvas requires a calculated height to fill available space on the page we have to subtract all heights that are not canvas: 

1em padding-top on .PlayDetailsTab_main__3Tcpx > div > div
1em padding-bottom on .PlayDetailsTab_main__3Tcpx > div > div
1 var(--bootstrap-padding) padding-top on .PlayDetailsTab_main__3Tcpx
1 var(--bootstrap-padding) padding-bottom on .PlayDetailsTab_main__3Tcpx
1 var(--bootstrap-padding) padding-top on .sticky-inner-wrapper > span
4em min-height on .sticky-inner-wrapper > span
2em min-height on .PlayDetailsNav_main__2F2BF

And leave at least one pixel to trigger the sticky header – that's the "-1px".

Reply:
And why is it `-3px` on wider displays?

Reply:
Those account for .sticky-outer-wrapper sometimes miscalculating height for unknown reasons.